800.renames-and-merges/python.yaml

```
@@ -291,7 +291,9 @@
 - { setname: "python:pysha3",          name: pysha3 } # CVE
 - { setname: "python:pyside",          name: pyside }
 - { setname: "python:pyside",          name: [pyside-qt4,pyside-qt4-py3], addflavor: true }
-- { setname: "python:pyside2",         name: pyside2 }
+- { setname: "python:pyside-tools",    name: pyside-tools }
+- { setname: "python:pyside2",         namepat: "(?:python:)?pyside-?2(-tools)?", addflavor: $1 }
+- { setname: "python:pyside6",         namepat: "(?:python:)?pyside-?6(-tools)?", addflavor: $1 }
```

I could do:

```suggestion
- { setname: "python:pyside",         namepat: "(?:python:)?pyside-?[0-9]+(-tools)?", addflavor: $1 }
```

and then allow `950.split-branches.yml` to handle it so that future pysides are all in one place, if that makes sense? Particularly given that we need the 950 rules for WikiData, Homebrew etc anyway.